src/graph.js

```diff
@@ -11,20 +11,20 @@ class Graph {
 	 * Creates a client to a specific graph running on the specific host/post
 	 * See: node_redis for more options on createClient
 	 *
-	 * @param graphId the graph id
-	 * @param host Redis host or node_redis client
-	 * @param port Redis port
-	 * @param options node_redis options
+	 * @param {string} graphId the graph id
+	 * @param {*} host Redis host or node_redis client
+	 * @param {*} port Redis port
```

Comment:
why `{*}` I guess you would like to mark as optional

Reply:
fixed